### ISSUE TITLE
Recognize the Alberta Elders' Cree Dictionary (AE) as the third Cree dictionary source

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#495]
Ensure that the dictionary XML importer recognizes `AE` as the code for the Alberta Elders' Cree Dictionary (AECD), and that the importer is able to handle (at least) three sources, even if with only a token case for `AE`, for now.

Edit: fixed reference code to `AE`.

**Comment by aarppe** [comment]
The latest crkeng.xml in the ALTLab github repo now has specifications for EC, and two example cases, for _wâpamêw_ and _wâpahtam_:

```
  <source id="EC">
      <title>Alberta Elders' Cree Dictionary</title>
   </source>
```

```
<e>
   <lg>
      <l pos="V">wâpamêw</l>
      <lc>VTA-1</lc>
      <stem>wâpam-</stem>
   </lg>
   <mg>
   <tg xml:lang="eng">
       <t pos="V" sources="MD EC">He sees him.</t>
   </tg>
   </mg>
   <mg>
   <tg xml:lang="eng">
       <t pos="V" sources="CW">s/he sees s.o., s/he witnesses s.o.</t>
   </tg>
   </mg>
   <mg>
   <tg xml:lang="eng">
       <t pos="V" sources="EC">S/he sees him/her.</t>
   </tg>
   </mg>
</e>
```

```
<e>
   <lg>
      <l pos="V">wâpahtam</l>
      <lc>VTI-1</lc>
      <stem>wâpaht-</stem>
   </lg>
   <mg>
   <tg xml:lang="eng">
       <t pos="V" sources="MD EC">He sees it.</t>
   </tg>
   </mg>
   <mg>
   <tg xml:lang="eng">
       <t pos="V" sources="CW">s/he sees s.t., s/he witnesses s.t.</t>
   </tg>
   </mg>
</e>
```

**Comment by aarppe** [comment]
@Madoshakalaka This is not necessarily the most urgent issue, but even if it will take a while for us to implement the integration of EC, our importer should be ready for that content.

**Comment by aarppe** [comment]
This appears to have been implemented in #488 and associated updates.

**Comment by aarppe** [comment]
@Madoshakalaka Does `crkeng.xml `get imported automatically upon updates, or does it still require a manual invocation? (I presume it's yet manual, but you were talking about turning the process automatic.)
